tasks/go.py

```diff
@@ -44,9 +45,19 @@ def lint(ctx, targets=None):
     targets_list = ["{}/...".format(t) for t in targets_list]
     result = ctx.run("golint {}".format(' '.join(targets_list)))
     if result.stdout:
-        files = {x for x in result.stdout.split('\n') if x}
-        print("Linting issues found in files: {}".format(','.join(files)))
-        raise Exit(1)
+        files = []
+        skipped_files = []
+        for x in ( y for y in result.stdout.split('\n') if y ):
+            fname = os.path.basename(x.split(":")[0])
+            if fname in lint_whitelist:
+                if not fname in skipped_files:
```

Comment:
not sure why you expect duplicates here but let's make `skipped_files` a set and always do the insertion so we can remove this line and make the code more readable